custom/icds_reports/tasks.py

```diff
@@ -1838,8 +1838,8 @@ def reconcile_data_not_in_ucr(reconciliation_status_pk):
             num_docs_unporcessed += 1
 
         log = {
-            "doc_id": doc_id, "sql_modified_on": sql_modified_on, "in_es": found_in_es,
-            "subtype": doc_subtype
+            "doc_id": doc_id, "modified_on": sql_modified_on.isoformat(), "inserted_at": inserted_at.isoformat(),
```

Comment:
nit: i think this would be more readable if each key was on its own line. It is a convention I have come to really like